doc/index.rst

```diff
@@ -36,8 +36,7 @@ Datasets for examples
    :toctree: generated/
    :template: function.rst
 
-   datasets.fetch_midwest_survey
-   datasets.fetch_employee_salaries
+   datasets.fetching.fetch_employee_salaries
```

Comment:
is it coherent to only show fetch_employee_salaries? should we show also midwest survey (that is also used for examples)? Or none of those?